### PR TITLE
Can no longer vote for crash in high pop

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Crash pop limited to 40, like it was before

## Why It's Good For The Game

Crash is ew

## Changelog
:cl:
fix: Cannot vote for crash when pop is > 40
/:cl:

### CODE REVIEW

**Comment by ilikesaying** [comment]
fuk u i like crash

**Comment by BraveMole** [comment]
> fuk u i like crash

that's because you are cringe

**Comment by bobmancer343** [comment]
speedmerge now

File: code/datums/gamemodes/game_mode.dm [hunk 1]
```diff
@@ -4,6 +4,7 @@
 	var/votable = TRUE
 	var/probability = 0
 	var/required_players = 0
+	var/maximum_players = INFINITY
```
Comment (TiviPlus) [inline code/datums/gamemodes/game_mode.dm:7]: Why is this not in config

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Review by TiviPlus — APPROVED** [review]
Tested?

**Comment by TiviPlus** [comment]
image
wait why is this a dm file

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**